### PR TITLE
fix #66 random text response

### LINKED ISSUE #66: Random text query error
**Description of the bug:**

Response of random text is not taken care of. Response JSON of random text returns an empty array of 'answers' key, as below :

json

This results in the following error :

bug

It is expected that the app does not stop abruptly and returns a sorry response, like 'Could not get you' or any appropriate message.

I would like to work on this issue.

### PR DESCRIPTION
**Pull request by abhisharma404**
The random text query bug is fixed, this is the result : 

fixed

### CODE REVIEW

File: python_wrapper/susi_python/response_parser.py [hunk 1]
```diff
@@ -25,26 +25,28 @@ def get_action(jsn):
...
-
     try:
         identity_json = parsed_dict['session']['identity']
         identity = Identity(identity_json)
         session = Session(identity)
     except KeyError:
         session = None
-
-    return QueryResponse(answer,parsed_dict, session)
+    if parsed_dict['answers']:
```
Comment (prateekiiest) [inline python_wrapper/susi_python/response_parser.py:34]: Check if the len of parse_dict[answers] is not zero
and update accordingly

**Comment by prateekiiest** [comment]
Please mention the issue you are working on by linking to the issue number

**Review by sansyrox — CHANGES_REQUESTED** [review]
@abhisharma404 , please create / refer the issue that you are solving prior to making a PR

**Comment by mariobehling** [comment]
@abhisharma404 Which issue are you solving? Where is the bug reported?

**Comment by prateekiiest** [comment]
The corresponding issue is this https://github.com/fossasia/susi_api_wrapper/issues/66 @mariobehling

**Comment by abhisharma404** [comment]
I'm sorry I was a bit busy with my exams. The issue that I had worked on is #66.
P.S. I'm new to git...so please bare my trivial questions! :)

File: python_wrapper/susi_python/response_parser.py (hunk 1 above)
Comment (abhisharma404) [reply python_wrapper/susi_python/response_parser.py:34]: This returns false if it's empty, it works either way. Should I change it?

**Review by sansyrox — CHANGES_REQUESTED** [review]
@prateekiiest @abhisharma404  , I believe this change will cause errors in SUSI Linux . As we expect an empty response there.

**Comment by prateekiiest** [comment]
I believe you are right @stealthanthrax

**Comment by mariobehling** [comment]
Should this be closed then?

**Comment by prateekiiest** [comment]
Due to the current breaking, I think we cant incorporate this.

**Comment by Orbiter** [comment]
rejected by developers

**Pull request closed without merging**